bootstrap/api/endpoint_test.go

```diff
@@ -204,6 +204,17 @@ func newThingsServer(csvc clients.Service, gsvc groups.Service, psvc tpolicies.S
 	capi.MakeHandler(csvc, mux, logger)
 	gapi.MakeHandler(gsvc, mux, logger)
 	papi.MakeHandler(csvc, psvc, mux, logger)
+
+	// Define a custom route to redirect /things/policies to /policies
+	mux.HandleFunc("/things/policies", func(w http.ResponseWriter, req *http.Request) {
+		http.Redirect(w, req, "/policies", http.StatusPermanentRedirect)
+	})
```

Comment:
This redirect seems to be unnecessary. Also, these redirects seem to only be relevant for changing state to inactive. Can we avoid them?

Reply:
How would we go about that, plus they are used for test?
Currently, the SDK assumes we are behind a gRPC endpoint which will route traffic from `things/policies` to `/policies` in the things container.

Reply:
You can do it by fixing the SDK to use the correct API: https://github.com/rodneyosodo/mainflux/blob/656c08417fc125646a95a5522d8c0f6bc1b3bb84/pkg/sdk/go/policies.go#L14

Reply:
>  Also, these redirects seem to only be relevant for changing state to inactive. Can we avoid them?

We use the SDK to also fetch a thing and channel hence modifying the thing URL to have a suffix `/things` will fail for fetching a thing and channel but will work for change state